.github/workflows/actions.yml

```diff
@@ -178,3 +178,86 @@ jobs:
           memory: max
       - name: ci/minikube/test-conbench-on-mk.sh
         run: export CONBENCH_REPO_ROOT_DIR=$(pwd) && bash ci/minikube/test-conbench-on-mk.sh
+
+  deploy-packages:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push'
```

Comment:
I always forget so many things about GitHub Actions. Does this here take effect only for merges to main? If yes -- how does the 'branch filtering' work?

Reply:
Yes, and that's because of this configuration:
https://github.com/conbench/conbench/blob/411c8132c6a9936e45473602f35c41a528172110/.github/workflows/actions.yml#L4-L6
I'll add a code comment!